docs/release/release-tasks.md

```diff
@@ -360,15 +360,23 @@ The goal of this task is to keep the CAPI community updated on recent PRs that h
 
 #### Create PR for release notes
 1. Checkout the `main` branch.
-1. Generate release notes with:
+2. Generate release notes with:
 
+   1. RELEASE CANDIDATE/BETA RELEASE example:
    ```bash
    # RELEASE_TAG should be the new desired tag (note: at this point the tag does not yet exist).
+   # PREVIOUS_VERSION_TAG is the previous released tag for determining the changes.
```

Comment:
How would this work for the first pre-release version without a pre-release precedent? Is that worth documenting as well?

Reply:
Updated the docs for the same, a minor code change was also required for it.